build/Makefile

```diff
@@ -163,10 +163,21 @@ ifdef NO_RACE_DETECTOR
   RACE_DETECTOR_ARGS=
 endif
 
-go_test_cmd = go test -mod=vendor $(RACE_DETECTOR_ARGS)
+go_test_args = -mod=vendor $(RACE_DETECTOR_ARGS)
+go_test_cmd = go test $(go_test_args)
 GO_TEST=$(DOCKER_RUN) $(go_test_cmd)
 GO_E2E_TEST_ARGS=--kubeconfig /root/.kube/$(kubeconfig_file)
 
+gotestsum_json=/tmp/agones.gotestsum.json
+ifdef GOTESTSUM_VERBOSE
+gotestsum_format=--format=testname \
+  --post-run-command="sh -c 'echo; echo --- RAW VERBOSE OUTPUT ---; jq -j \"select(.Output != null) | .Output\" < $(gotestsum_json); echo --- END RAW VERBOSE OUTPUT ---'"
+else
+gotestsum_format=--format=standard-quiet
+endif
+gotestsum_cmd=gotestsum $(gotestsum_format) --jsonfile=$(gotestsum_json) --hide-summary=skipped --rerun-fails=2 $(gotestsum_post_run_command) $(GOTESTSUM_ARGS)
```

Comment:
Is `gotestsum_post_run_command` set anywhere? Is it supposed to be the same thing as the `--post-run-command` in line 174?

Reply:
Is `GOTESTSUM_ARGS` needed? You mentioned "use `GOTESTSUM_ARGS` to pass args to gotestsum (e.g. --rerun-fails=5)", but looks like those fields (i.e. jsonfile, hide-summary, rerun-failes) are set explicitly here.

Reply:
`gotestsum_post_run_command` was a vestigal of a previous factoring - missed that, thanks. Removed.

The idea behind `GOTESTSUM_ARGS` was just as an entry point to allow the user to specify more args specifically for `gotestsum`. We default to the ones you mention, but I intentionally but it at the end so if you want to override any of those default things (e.g. `--rerun-fails=5`), you can do that.